src/app/dashboard-widgets/recent-pipelines-widget/recent-pipelines-widget.component.ts

```diff
@@ -29,14 +29,8 @@ export class RecentPipelinesWidgetComponent implements OnInit, OnDestroy {
     this.bcs = this.pipelinesService.recentPipelines
       .publish();
     this.buildConfigs = this.bcs;
```

Comment:
What's the purpose of holding `buildConfigs` and `bcs` variables? While we're here can one be dropped and the other used in all places? `bcs` is of type `any`, that seems strange.